optimism/src/mips/interpreter.rs

```diff
@@ -805,7 +881,37 @@ pub fn interpret_rtype<Env: InterpreterEnv>(env: &mut Env, instr: RTypeInstructi
             return;
         }
         RTypeInstruction::JumpAndLinkRegister => (),
-        RTypeInstruction::SyscallMmap => (),
+        RTypeInstruction::SyscallMmap => {
+            let requested_alloc_size = env.read_register(&Env::constant(5));
+            let size_in_pages = {
```

Comment:
I don't understand this. Why `instruction`?

Reply:
Oopsie! Fixed